qutebrowser/mainwindow/tabwidget.py

```diff
@@ -705,7 +705,17 @@ def wheelEvent(self, e):
             e: The QWheelEvent
         """
         if config.val.tabs.mousewheel_switching:
-            super().wheelEvent(e)
+            if utils.is_mac:
```

Comment:
Please add a comment such as 

```suggestion
            if utils.is_mac:
                # WORKAROUND for this not being customizable until Qt 6:
                # https://codereview.qt-project.org/c/qt/qtbase/+/327746
```

so we know there's a more native solution for this issue with Qt 6.